sdks/python/apache_beam/__init__.py

```diff
@@ -69,7 +69,7 @@
 if not (sys.version_info[0] == 2 and sys.version_info[1] == 7):
   raise RuntimeError(
       'Dataflow SDK for Python is supported only on Python 2.7. '
-      'It is not supported on Python [%s].' % sys.version_info)
+      'It is not supported on Python '+ str(sys.version_info) + '.')
```

Comment:
Do you want to put `[]` around the version or remove the `.` at the end to avoid confusion?